File: algorithm/src/main/java/baekjoon/p10768/Main.java
package baekjoon.p10768;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println(solve(sc.nextInt(), sc.nextInt()));
    }

    private static String solve(int a, int b) {
        if (a < 2) {
            return "Before";
        } else if (a > 2) {
            return "After";
        } else {
            if (b < 18) {
                return "Before";
            } else if (b > 18) {
                return "After";
            } else {
                return "Special";
            }
        }
    }

}

File: algorithm/src/main/java/baekjoon/p10768/Main.kt
package baekjoon.p10768

fun main(args: Array<String>) {
    println(solve(readLine()!!.toInt(), readLine()!!.toInt()))
}

private fun solve(a: Int, b: Int): String {
    return when {
        a < 2 -> "Before"
        a > 2 -> "After"
        else -> when {
            b < 18 -> "Before"
            b > 18 -> "After"
            else -> "Special"
        }
    }
}